src/components/pages/LogoutPage.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from '@mui/material/Link';
+import { NavLink } from 'react-router-dom'
+import { Container, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import viewerState from '../../state/ViewerState';
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+interface LogoutPageProps {
+    isLoggedin: boolean
+}
+
+const LogoutPage: React.FC<LogoutPageProps> = ({ isLoggedin }) => {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+
+    const [errorMessage, setErrorMessage] = useState<string | null>('');
+
+    const handleLogout = async () => {
+        try {
+            if (isLoggedin) {
+                let response = await axios.post('http://localhost:8000/logout/', null, {
+                    headers: {
```

Comment:
Would be good to move all the localhost:8000 into one place in settings

Reply:
I agree, we should not be manually edit this everywhere everytime it changes.

I was not sure what you meant by settings (I know we have a settings.py file in the backend, but I am not sure about if we have similar in the frontend), so I created a "urlHelpers" file with functions to retrieve automatically protocol, hostname and everything related to the URL. Assuming the backend is in the same host, with a different port, I created a function with the port hardcoded, but that fills everything else automaticallyu (maybe we can convert the port and hostname into an environment variable and we could set an environment or config file?).